plane/source/docs/guide-esc-calibration.rst

```diff
@@ -0,0 +1,69 @@
+.. _guide-esc-calibration:
+
+===============
+ESC Calibration
+===============
+
+If your plane uses a ESC (electronic speed control) for motor output
+then you will probably need to calibrate it.
+
+.. warning:: You should remove the propeller from your aircraft before
+             starting ESC calibration
+
+Calibrating your ESC involves teaching your ESC what range of throttle
+inputs it should respond to. It needs to know what PWM value on the
+throttle channel corresponds with the commanding the motor to be off,
+and what PWM value corresponds with full throttle. These values are
+stored inside the ESC.
+
+.. note:: You should not try to calibrate your ESC until you have
+          completed both your :ref:`RC inputs <rc-throw-trim>`
+          calibration and your :ref:`servo setup <servo-functions>`.
+
+Typical ESC Calibration
+=======================
+
+The most common type of ESC calibration for small electric aircraft is
+the max-throttle/min-throttle method.
+
+To perform this type of calibration you will need to be able to
+control power to your motor separately from power to the autopilot. If
+you don't have separate power, then you can temporarily power your
+autopilot from a USB cable for the purposes of this calibration.
+
+The steps for calibration are:
+
+- start with the autopilot powered, but with the motor unpowered
+- switch to MANUAL flight mode
+- disable the safety on outputs
```

Comment:
Here's the "safety on outputs" term again. Are you referring to the switch? I don't know what this is and doubt anyone else does too